python/pynessie/client/nessie_client.py

```diff
@@ -193,17 +193,16 @@ def get_content(self: "NessieClient", ref: str, content_key: ContentKey, hash_on
 
         return ContentSchema().load(get_content(self._base_url, self._auth, ref_name, content_key, hash_on_ref, self._ssl_verify))
 
-    def commit(
-        self: "NessieClient", branch: str, old_hash: str, reason: Optional[str] = None, author: Optional[str] = None, *ops: Operation
-    ) -> Branch:
+    # pylint: disable=keyword-arg-before-vararg
```

Comment:
this is a bad pattern but we cant fix it here since this method is part of the public api, so we ignore the rule locally